src/mbgl/renderer/render_layer.cpp

```diff
@@ -43,7 +43,7 @@ RenderLayer::RenderTiles RenderLayer::filterRenderTiles(RenderTiles tiles) const
 }
 
 void RenderLayer::sortRenderTiles(const TransformState&) {
-    std::sort(renderTiles.begin(), renderTiles.end(), [](const auto& a, const auto& b) { return a.get().id < b.get().id; });
+    // no-op
```

Comment:
If this function is a no-op, we can remove it entirely.

Reply:
`RenderSymbolLayer` still overrides it, however this is something I'd like to change in the following PR.